src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -76,6 +76,30 @@ public function get($uri, array $headers = [])
     }
 
     /**
+     * Visit the given URI with a GET request with content type of application/json.
+     *
+     * @param  string  $uri
+     * @param  array  $data
+     * @param  array  $headers
+     * @return $this
+     */
+    public function getJson($uri, array $data = [], array $headers = [])
+    {
+        $content = json_encode($data);
+        $headers['CONTENT_TYPE'] = 'application/json';
+        $headers['CONTENT_LENGTH'] = mb_strlen($content);
```

Comment:
That's unfortunately all we have to work with. The number of bytes to expect *is* the length of the string, and I'm unaware of any way in PHP to find the number of bytes in a string other than getting it's length.